File: 卡码/代码随想录/1/04-字符串/C++/01-反转字符串/LeetCode_0344.cpp
//344. ·´×ª×Ö·û´®
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
  void reverseString(vector<char>& s) {
		int left=0,right=s.size()-1;
		while(left<right){
			s[left]^=s[right];
			s[right]^=s[left];
			s[left++]^=s[right--];
		}
  }
};

File: 卡码/代码随想录/1/04-字符串/Java/src/_01_反转字符串/LeetCode_0344.java
//344. 反转字符串
package _01_反转字符串;

public class LeetCode_0344 {

}
class Solution {
  public void reverseString(char[] s) {
  	int left=0,right=s.length-1;
  	while(left<right) {
  		s[left]^=s[right];
  		s[right]^=s[left];
  		s[left++]^=s[right--];
  	}
  }
}
